scout/adapter/mongo/event.py

```diff
@@ -210,6 +208,33 @@ def events(
 
         return self.event_collection.find(query).sort("created_at", pymongo.DESCENDING)
 
+    def events_by_variant_id(
+        self,
+        variant_id,
+        verb=None,
+        institute=None,
+    ):
+        """Return all events for variant_id, optionally selected for verb.
+        Note that this is across all cases. Otherwise use events().
+
+        Args:
+            variant_id: md5 hash simple id chr_pos_ref_alt
```

Comment:
There are some missing args here, perhaps you modified the function at a later stage?